ignite/__init__.py

```diff
@@ -1 +1,3 @@
+from .engines.engine import Engine, Events, State
```

Comment:
Good idea to add principal classes. Maybe you can add handlers and metrics ?
Anyway, import should be absolute and `engineS` is `engine` now :)

Reply:
Thanks, fixed. You'd like to add all the handlers here?